src/index.js

```diff
@@ -2,14 +2,14 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import { Provider } from 'react-redux';
-import App from './App';
+import Home from './pages/Home';
 import store from './app/store';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
   <React.StrictMode>
     <Provider store={store}>
-      <App />
+      <Home />
     </Provider>
```

Comment:
Why did you delete the App component and rendered your Homepage component here? Do you think that we need to delete the App, if yes why do you think that we shouldn't render our components in the App but in index.js directly ?

Reply:
I forget to delete it :( gonna update